setup.cfg

```
@@ -100,10 +100,16 @@ doctest_plus = enabled
 text_file_format = rst
 open_files_ignore = "astropy.log" "/etc/hosts" "*.ttf"
 remote_data_strict = true
-addopts = -p no:warnings --doctest-rst
+addopts = --doctest-rst
 asdf_schema_root = astropy/io/misc/asdf/data/schemas
 xfail_strict = true
 qt_no_exception_capture = 1
+filterwarnings =
```

When we think we have addressed all the remaining warnings, 🔥 
```suggestion
filterwarnings =
    error
```